terracumber_config/tf_files/SUSEManager-Tests-Reporting-Hub.tf

```diff
@@ -0,0 +1,221 @@
+    hostname = "suma-reportdb-hub-srv.mgr.suse.de"
+  }
+}
+
+module "slave1" {
+  source = "./modules/server"
+  base_configuration = module.base_core.configuration
+  product_version = "head"
+  name = "slave1"
+  additional_repos = {
+    Test_repo = "http://download.suse.de/ibs/Devel:/Galaxy:/Manager:/TEST:/Orion/SLE_15_SP4/"
+  }
+  auto_accept                    = true
+  use_os_released_updates        = false
+  from_email                     = "root@suse.de"
+  register_to_server = module.hub-server.configuration.hostname
+  image = "sles15sp4o"
+  provider_settings = {
+    mac = "aa:b2:93:01:00:e2"
+  }
+  server_configuration = {
+    hostname = "suma-reportdb-slave1.mgr.suse.de"
+  }
+}
+
+module "slave2" {
+  source = "./modules/server"
+  base_configuration = module.base_core.configuration
+  product_version = "head"
+  name = "slave2"
+  additional_repos = {
+    Test_repo = "http://download.suse.de/ibs/Devel:/Galaxy:/Manager:/TEST:/Orion/SLE_15_SP4/"
+  }
+  auto_accept                    = false
+  use_os_released_updates        = false
+  from_email                     = "root@suse.de"
+  register_to_server = module.hub-server.configuration.hostname
+  image = "sles15sp4o"
+  provider_settings = {
+    mac = "aa:b2:93:01:00:e3"
+  }
+  server_configuration = {
+    hostname = "suma-reportdb-slave2.mgr.suse.de"
+  }
+}
+
+
+module "min-sles15sp3" {
+  source = "./modules/minion"
+  base_configuration = module.base_core.configuration
+  name = "min-sles15sp3"
+  image = "sles15sp3o"
+  server_configuration = module.slave1.configuration
+  use_os_released_updates = false
+  provider_settings = {
+    mac = "aa:b2:93:01:00:e4"
+  }
+  server_configuration = {
+    hostname = "suma-reportdb-min-sles15sp3.mgr.suse.de"
+  }
+}
+
+module "min-centos7" {
+  source = "./modules/minion"
+  base_configuration = module.base_core.configuration
+  name = "min-centos7"
+  image = "centos7o"
+  server_configuration = module.slave1.configuration
+  use_os_released_updates = false
+  provider_settings = {
+    mac = "aa:b2:93:01:00:e5"
+  }
+  server_configuration = {
+    hostname = "suma-reportdb-min-centos7.mgr.suse.de"
+  }
+}
+
+module "controller" {
+  source = "./modules/controller"
+  base_configuration = module.base_core.configuration
+  name = "ctl"
+  no_auth_registry = "registry.mgr.suse.de"
+  auth_registry = "portus.mgr.suse.de:5000/cucutest"
+  auth_registry_username = "cucutest"
+  auth_registry_password = "cucusecret"
+
+  git_username = var.GIT_USER
+  git_password = var.GIT_PASSWORD
+  git_repo     = var.CUCUMBER_GITREPO
+  branch       = var.CUCUMBER_BRANCH
+  git_profiles_repo = "https://github.com/uyuni-project/uyuni.git#:testsuite/features/profiles/internal_nue"
+
+  server_http_proxy = "galaxy-proxy.mgr.suse.de:3128"
+  
+  server_configuration = module.slave1.configuration
+
+  sle15sp3_minion_configuration = module.min-sles15sp3.configuration
+  centos7_minion_configuration = module.min-centos7.configuration
+
```

Comment:
Maybe not yet, but once we start adding Cucumber features about it.
We need to modify sumaform to include as variables the configuration of slave1 and slave2, or whatever name we want to use (peripheral?)